Dockerfile

```diff
@@ -1,6 +1,6 @@
 ### This is a generated file from Dockerfile.in ###
 #@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:rhel_8_golang_1.17)
```

Comment:
Do the `follow_tag` declarations need any update? Can we remove them?

Reply:
The `follow_tag` declarations are only used in downstream builds by brew. However to make upstream<->midstream easy we should keep them or else any the diff between them gets bigger. At least we should keep them in the `Dockerfile.in`